jboss-kie-smartrouter/tests/bats/resources/logging.properties

```diff
@@ -0,0 +1,5 @@
+handlers= java.util.logging.ConsoleHandler
```

Comment:
remove it and copy from jboss-kie-smartrouter/added/configuration/logging.properties